yupana/processor/report_slice_processor.py

```diff
@@ -200,11 +211,13 @@ def generate_upload_candidates(self):
                       for key in host.keys() if key not in ['cause', 'status_code']}
         return candidates
 
-    def _transform_tags(self, host: dict):
+    def _transform_tags(self, host: dict, transformed_obj=None):
```

Comment:
Can you call to `transformed_dict` directly?
```suggestion
    def _transform_tags(self, host: dict, transformed_obj=self.transformed_dict()):
```
it will remove the need for internal `if`transformed_obj is None` statement

Reply:
Yes, I tried in this way earlier. With this change, it is giving me below warning- 

`pylint: warningW0102 - Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)`

Reply:
Maybe we can use a const value to remove the warning? Or even duplicate the value of a const, so we won't change the original dictionary?

Reply:
@ShimShtein - Done! Fixed by making use of `copy` module. Please let me know your thoughts on changes.